### PR TITLE
feat: add sort by sn for drop page

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature

## Context

- [x] Related #9669

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

## Screenshot 📸

- [x] My fix has changed UI

Screenshot 2024-03-08 at 09 11 03

### CODE REVIEW

File: utils/constants.ts [hunk 1]
```diff
@@ -89,6 +89,8 @@ export const NFT_SQUID_SORT_CONDITION_LIST: string[] = [
   'updatedAt_ASC',
   'price_ASC',
   'price_DESC',
+  // 'sn_ASC',
+  // 'sn_DESC',
```
Comment (Jarsen136) [inline utils/constants.ts:93]:
```suggestion
```

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
you added it only on drops, nice! 
@preschian is it hard to make Serial -> High - Low the default instead of newest?

**Comment by preschian** [comment]
> is it hard to make Serial -> High - Low the default instead of newest?

Unfortunately, for now, it is not so straightforward; I need to exclude it from some pages. unless fixed on the indexer side
- https://github.com/kodadot/nft-gallery/pull/9551#issuecomment-1963555392
- https://github.com/kodadot/nft-gallery/pull/9551#issuecomment-1964441579

**Comment by prury** [comment]
> > is it hard to make Serial -> High - Low the default instead of newest?
> 
> Unfortunately, for now, it is not so straightforward; I need to exclude it from some pages. unless fixed on the indexer side
> - https://github.com/kodadot/nft-gallery/pull/9551#issuecomment-1963555392
> - https://github.com/kodadot/nft-gallery/pull/9551#issuecomment-1964441579

Let's procced then with the current solution np! Thank you

**Comment by yangwao** [comment]
what's about tests?

image

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 10.35 USD/DOT ~ 2.899 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0x7b2b075119e0255fd247d3016ac70c22f2ead4549bf4abdc129b3b89e11c57fd](https://polkadot.subscan.io/extrinsic/0x7b2b075119e0255fd247d3016ac70c22f2ead4549bf4abdc129b3b89e11c57fd)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**